pkg/translate/translate_value.go

```diff
@@ -244,18 +247,6 @@ func (t *ReverseTranslator) setEnablementAndNamespacesFromValue(valueSpec map[st
 		}
 	}
 
-	// set namespace
```

Comment:
Is this moved somewhere? I'm not sure if we can just remove it.

Reply:
it is done during the API mapping already